pydra/engine/tests/test_nipype1_convert.py

```diff
@@ -119,4 +119,4 @@ def test_interface_run_1():
     task = TouchInterf(new_file="hello.txt")
     assert task.cmdline == "touch hello.txt"
     res = task()
-    assert res.output.new_file.exists()
+    assert res.output.new_file.fspath.exists()
```

Comment:
I wonder if `File` could also have `exists` methods?

Reply:
fspaths in File are actually already checked for existence, so this check is redundant. I just kept it there so that new_file is checked to be a File essentially. Probably a more elegant way to do this but I was trying to keep the tests as similar as possible